pkg/distro/fedora/distro.go

```diff
@@ -726,6 +696,14 @@ func newDistro(version int) distro.Distro {
 	}
 	vhdImgType.defaultImageConfig = vhdConfig.InheritFrom(qcow2ImgType.defaultImageConfig)
 
+	minimalrawZstdImgType := minimalrawImgType
```

Comment:
It's interesting that the image type name is "minimal-raw-zst", but the variable name is "minimalRawZst**d**ImgType". My gut feeling is that the d shouldn't be there.